README.md

```diff
@@ -1 +1,54 @@
-## To be updated
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+To Do List
+
+ This Projects involves building a clone of [minimalist](https://web.archive.org/web/20180320194056/http://www.getminimalist.com:80/) to do list.
+
+## Built With
+
+HTML, CSS & JS
+
+## [Live demo](https://devmunyi.github.io/Awesome-books-with-ES6/)
+
```

Comment:
Great job @devMunyi 🤝

- [ ] [Optional] Kindly remove this Live Demo link from your Readme file as this Link is for another project.